src/modules/UI.js

```diff
@@ -0,0 +1,78 @@
+import ListTemplate from './Class.js';
+import localGet from './Store.js';
+
+const todoContent = document.getElementById('listItem');
+const todoTask = document.getElementById('input');
+const displayList = () => {
+  todoContent.replaceChildren();
+  localGet().forEach((item, id) => {
+    let isCompleted;
+    if (item.completed === true) {
+      isCompleted = 'checked';
+    } else {
+      isCompleted = '';
+    }
+
+    todoContent.innerHTML += `
+      <div class="toDoItem">
+        <div class="added-checkbox">
+
+        <input class='item' id='check-${id}', "completed")' type='checkbox' ${item.Checked ? 'true' : 'false'} onChange='updateList(${id}, "completed")' ${isCompleted}>
+        <input onkeyup='updateList(${id})' type="text" class='findInput' id='input-${id}' value='${item.description}' />
+
+        </div>
+        <i class="delete" onclick='removeList(${item.index})' id='delete-${item.index}'>🗑️</i>
+      </div>
+        `;
+  });
+};
+
+const addList = (description, completed, index) => {
+  const listAdded = new ListTemplate(description, completed, index);
+  const x = localGet();
+  x.push(listAdded);
+  localStorage.setItem('listStorage', JSON.stringify(x));
+  todoTask.value = '';
```

Comment:
You can create a function to set items as you are using this at multiple places in code

Reply:
Thanks for the review. I have done the required changes.